synths/default-synths.scd

```diff
@@ -124,7 +124,7 @@ SynthDef(\tutorial4, {|out, begin=0, end=1, pan, speed=1, accelerate, n |
 }).add;
 
 // finally, as an example of how to add your own parameters, let's say we want precise control over frequency
-// we'll make a new "f" parameter instead of "n" (don't use "freq", it's already used internally in SuperDirt)
```

Comment:
This is fixed here: https://github.com/musikinformatik/SuperDirt/commit/60948262d871026fafc788ffbd7cc911f94d9580

I think we should use `freq` instead of `f`or event `n`(when `n.midicps` is done later), because it is easier to understand and also we can apply harmonic calculations in superdirt better.

What do you think?